textract/layout/utils.py

```diff
@@ -0,0 +1,49 @@
+from enum import Enum
+
+class LayoutType(Enum):
+	Page = 0
+	Paragraph = 1
+	Line = 2
+	Image = 3
+
+class LayoutUtils():
+	def __init__(self, id_, src, type_):
+		self._id   = id_
+		self._src  = src
+		self._type = type_
+
+	def get_id(self):
+		return self._id
+
+	def get_src(self):
+		return self._src
+
+	def get_type(self):
+		return self._type
+
+class Page(LayoutUtils):
+	def __init__(self, id_, src):
+		super().__init__(id_, src, LayoutType.Page)
+
+class Paragraph(LayoutUtils):
+	def __init__(self, id_, src):
+		super().__init__(id_, src, LayoutType.Paragraph)
+
+class Line(LayoutUtils):
+	def __init__(self, id_, src):
+		super().__init__(id_, src, LayoutType.Line)
+
+class Image(LayoutUtils):
+	def __init__(self, id_, src):
+		super().__init__(id_, src, LayoutType.Image)
+
```

Comment:
Personally, I'd more like to use `functools.partial` here. 
e.g.
```python
import functools

class LayoutUtils:
    def __init__(self, type, id, src):
        ....

Page = functools.partial(LayoutUtils, LayoutType.Page)
Paragraph = functools.partial(LayoutUtils, LayoutType. Paragraph)
Line = functools.partial(LayoutUtils, LayoutType. Line)
Image = functools.partial(LayoutUtils, LayoutType. Image)
```

Again, your code looks good, just for fun :)